bindata/manifests/cni-config/sriov-cni-config.yaml

```diff
@@ -25,6 +25,12 @@ spec:
 {{- end -}}
 {{- if .VlanQoSConfigured -}}
   "vlanQoS":{{.SriovCniVlanQoS}},
+{{- end -}}
+{{- if .MinTxRateConfigured -}}
+  "minTxRate":{{.SriovCniMinTxRate}},
+{{- end -}}
+{{- if .MaxTxRateConfigured -}}
+  "maxTxRate":{{.SriovCniMaxTxRate}},
```

Comment:
ditto, the key name of `maxTxRate` should be `max_tx_rate`

Reply:
Done. Thanks.